dev/io.openliberty.microprofile.reactive.streams.operators30.internal_fat_tck/publish/tckRunner/tck/src/test/java/org/eclipse/microprofile/reactive/streams/operators/tck/spi/OnErrorResumeStageVerification.java

```diff
@@ -0,0 +1,292 @@
+        throw new QuietRuntimeException("failed");
+      })
+      .toList()
+      .run(getEngine()));
+  }
+
+  @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = ".*failed.*")
+  public void onErrorResumeWithRsPublisherStageShouldPropagateRuntimeExceptions() {
+    await(rs.failed(new QuietRuntimeException("source-failure"))
+      .onErrorResumeWithRsPublisher(t -> {
+        throw new QuietRuntimeException("failed");
+      })
+      .toList()
+      .run(getEngine()));
+  }
+
+  @Test(expectedExceptions = QuietRuntimeException.class, expectedExceptionsMessageRegExp = ".*boom.*")
+  public void onErrorResumeWithShouldBeAbleToInjectAFailure() {
+    await(rs.failed(new QuietRuntimeException("failed"))
+      .onErrorResumeWith(err -> rs.failed(new QuietRuntimeException("boom")))
+      .toList()
+      .run(getEngine()));
+  }
+
+  @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = ".*boom.*")
+  public void onErrorResumeWithRsPublisherShouldBeAbleToInjectAFailure() {
+    await(rs.failed(new QuietRuntimeException("failed"))
+      .onErrorResumeWithRsPublisher(err -> rs.failed(new QuietRuntimeException("boom")).buildRs(getEngine()))
+      .toList()
+      .run(getEngine()));
+  }
+
+    @Override
+    List<Object> reactiveStreamsTckVerifiers() {
+        return Arrays.asList(
+            new OnErrorResumeWithVerification(),
+            new OnErrorResumeVerification(),
+            new OnErrorResumeWithPublisherVerification()
+        );
+    }
+
+    public class OnErrorResumeWithVerification extends StageProcessorVerification<Integer> {
+
+        @Override
+        public Processor<Integer, Integer> createIdentityProcessor(int bufferSize) {
+            return rs.<Integer>builder()
+                .onErrorResumeWith(rs::failed)
+                .map(Function.identity()).buildRs(getEngine());
+        }
+
+        @Override
+        public Publisher<Integer> createFailedPublisher() {
+            return rs.<Integer>failed(new RuntimeException("failed"))
+                .onErrorResumeWith(t -> {
+                    // Re-throw the exception.
+                    if (t instanceof RuntimeException) {
+                        throw (RuntimeException) t;
+                    }
+                    // Wrap if required.
+                    throw new RuntimeException(t);
+                })
+                .buildRs(getEngine());
+        }
+
+        @Override
+        public Integer createElement(int element) {
+            return element;
+        }
+        
+        /**
+         * 
+         * Liberty Change:
+         * Disable this test due to very infrequent intermittent failures
+         * See defect 286531
+         * 
+         */
+        @Override
+        @Test
+        public void required_spec109_mustIssueOnSubscribeForNonNullSubscriber() {
+            System.out.println("Not running required_spec109_mustIssueOnSubscribeForNonNullSubscriber");
+        }
```

Comment:
Since we've changed the underlying reactive streams implementation, we may not have the same problem here.

This class should be removed.